.eslintrc

```diff
@@ -45,7 +45,12 @@
     "no-console": 2,
     "no-unused-vars": [
       2,
-      { "args": "after-used", "argsIgnorePattern": "^_", "vars": "local" }
+      {
+        "args": "after-used",
+        "argsIgnorePattern": "^_",
+        "vars": "local",
+        "ignoreRestSiblings": true
```

Comment:
Adding `"ignoreRestSiblings": true` enabled a immutable way to remove a key value pair from an object will note below